ote/src/cljs/ote/views/passenger_transportation.cljs

```diff
@@ -186,7 +189,7 @@
 
 
 (defn passenger-transportation-info [e! {form-data ::t-service/passenger-transportation :as service}]
-  (with-let [form-groups
```

Comment:
why does the subtype change after initial render? is it in all cases?

Reply:
It don't change. I have no idea why this change affects as it does